mule-packager/src/main/java/org/mule/tools/api/packager/builder/MulePackageBuilder.java

```diff
@@ -28,12 +29,15 @@
 
 import org.codehaus.plexus.archiver.ArchiverException;
 
+import org.mule.tools.api.packager.archiver.Archiver;
+import org.mule.tools.api.packager.archiver.MuleArchiver;
+import org.mule.tools.api.packager.packaging.PackagingOptions;
 import org.mule.tools.api.packager.packaging.PackagingType;
 
 /**
  * Builder for Mule Application archives.
```

Comment:
Builder for Mule Application archives ->  Builder for Mule Application **packages**

It's confusing as this class doesn't really generate the archive